Lib/multiprocessing/managers.py

```diff
@@ -857,20 +864,20 @@ def _incref(self):
         dispatch(conn, None, 'incref', (self._id,))
         util.debug('INCREF %r', self._token.id)
 
-        self._idset.add(self._id)
+        self._idset.add(self._serial)
```

Comment:
How feasible is it to rename `self._idset` given it no longer contains Python `id()`s?

Reply:
Not too hard. `_serialset` looks less parseable so I'd go with `_all_serials`.